api/models/user.js

```diff
@@ -448,6 +448,33 @@ const updateUserEmail = (userId: string, email: string): Promise<Object> => {
     .then(() => getUserById(userId));
 };
 
+const deleteUser = (userId: string) => {
+  return db
+    .table('users')
+    .get(userId)
+    .update({
+      username: null,
+      email: null,
+      deletedAt: new Date(),
+      providerId: null,
+      fbProviderId: null,
+      googleProviderId: null,
+      githubProviderId: null,
+      githubUsername: null,
+      profilePhoto: null,
```

Comment:
Instead of showing no profile photo or the default avatar (which makes it seem like it's still an actual user), this should set the profile photo to something we created which makes it obvious that the user is deleted?

Reply:
Imo a placeholder avatar + "Deleted" as the name will be quite clear, right?